src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingOptInRequestsMobileWiki.java

```diff
@@ -0,0 +1,373 @@
+      dataProvider = "adsNordicsDataProvider",
+      groups = "AdsOptInRejectedMobileWiki"
+  )
+  public void adsTrackingRejectedForNordics(List<String> urlPatterns) {
+    networkTrafficInterceptor.startIntercepting();
+    TrackingOptInModal.setGeoCookie(driver, "EU", "DE");
+
+    new TrackingOptInModal()
+        .verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor, ADS_HOME_PAGE);
+  }
+
+  @NetworkTrafficDump(useMITM = true)
+  @Execute(trackingOptIn = false)
+  @Test(
+      dataProviderClass = TrackingOptInDataProvider.class,
+      dataProvider = "adsPrebidDataProvider",
+      groups = "AdsOptInRejectedMobileWiki"
+  )
+  public void adsTrackingRejectedForPrebid(String[] instantGlobals, List<String> urlPatterns) {
+    networkTrafficInterceptor.startIntercepting();
+    TrackingOptInModal.setGeoCookie(driver, "EU", "PL");
+
+    new TrackingOptInModal().verifyTrackingRequestsNotSend(instantGlobals, urlPatterns,
+                                                           networkTrafficInterceptor,
+                                                           ADS_HOME_PAGE);
+  }
+
+  @NetworkTrafficDump(useMITM = true)
+  @Execute(trackingOptIn = false)
+  @Test(
+      dataProviderClass = TrackingOptInDataProvider.class,
+      dataProvider = "adsA9DataProvider",
+      groups = "AdsOptInRejectedMobileWiki"
+  )
+  public void adsTrackingRejectedForA9(String[] instantGlobals, List<String> urlPatterns) {
+    networkTrafficInterceptor.startIntercepting();
+    TrackingOptInModal.setGeoCookie(driver, "EU", "PL");
+
+    new TrackingOptInModal().verifyTrackingRequestsNotSend(instantGlobals, urlPatterns,
+                                                           networkTrafficInterceptor,
+                                                           ADS_HOME_PAGE);
+  }
+
+  @NetworkTrafficDump(useMITM = true)
+  @Execute(trackingOptIn = false)
+  @Test(
+      dataProviderClass = TrackingOptInDataProvider.class,
+      dataProvider = "adsComscoreQuantcastDataProvider",
+      groups = "AdsOptInRejectedMobileWiki"
+  )
+  public void adsTrackingRejectedForComscoreAndQuantcast(List<String> urlPatterns) {
+    networkTrafficInterceptor.startIntercepting();
+    TrackingOptInModal.setGeoCookie(driver, "EU", "PL");
+
+    new TrackingOptInModal()
+        .verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor, ADS_HOME_PAGE);
+
+  }
+
+    @NetworkTrafficDump(useMITM = true)
+    @Execute(trackingOptIn = false)
+    @Test(
+            dataProviderClass = TrackingOptInDataProvider.class,
+            dataProvider = "adsGoogleAnalyticsDataProvider",
+            groups = "AdsOptInRejectedMobileWiki"
+    )
+    public void adsTrackingRejectedForGoogleAnalytics(List<String> urlPatterns) {
+        networkTrafficInterceptor.startIntercepting();
+        TrackingOptInModal.setGeoCookie(driver, "EU", "PL");  TrackingOptInModal.setGeoCookie(driver, "EU", "PL");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00122)